azurerm/resource_arm_netapp_pool_test.go

```diff
@@ -236,7 +236,7 @@ resource "azurerm_netapp_pool" "test" {
   location            = "${azurerm_resource_group.test.location}"
   resource_group_name = "${azurerm_resource_group.test.name}"
   service_level       = "Standard"
-  size_in_4_tb        = "2"
+  size                = "8"
```

Comment:
Should we have a test with a non multiple of 4?

Reply:
Swapped to `15`
```
--- PASS: TestAccAzureRMNetAppPool_basic (101.94s)
--- PASS: TestAccAzureRMNetAppPool_complete (106.63s)
--- PASS: TestAccAzureRMNetAppPool_update (123.98s)
```